include/astshim/PolyMap.h

```diff
@@ -64,6 +64,8 @@ class PolyMap : public Mapping {
         for details.
     @param[in] options  Comma-separated list of attribute assignments.
 
+    @throw std::invalid_argument if neither transform is specified (coeff_f and coeff_i are both empty)
+
```

Comment:
`@throw` is valid and unfortunately I've been using both. Is it worth standardizing?

Why did you bring up "if" being the first word of the description? I know that it is.